shoop/core/settings.py

```diff
@@ -136,3 +136,9 @@
 
 #: The submission URL for Shoop's telemetry (statistics) system
 SHOOP_TELEMETRY_URL = "https://telemetry.shoop.io/collect/"
+
+#: Default cache duration for various caches in seconds
+SHOOP_DEFAULT_CACHE_DURATION = 60 * 30
```

Comment:
Another point: I really don't want people to have to import `datetime` in their `settings` for this.